### PR TITLE
Recaptcha v3 publishing

### PR DESCRIPTION
**Pull request by PedroAntunesCosta**
This PR promotes 2 beta reCAPTCHA guides to the main published articles and also publishes the corresponding release note. This is meant for the reCAPTCHA v3 support launch in 16/06/2023.

#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

### CODE REVIEW

File: docs/guides/Integration-Guides/recaptcha/implementing-recaptcha-in-integrations.md [hunk 1]
```diff
@@ -3,22 +3,73 @@ title: "Implementing reCAPTCHA in integrations"
...
+            "score": "{recaptcha-v3-score}"
+        }
+    ],
+    …
+}
+```
+
+- `recaptchaKeys`: array of objects with keys information.
+- `clientId`: ID of your reCAPTCHA key.
+- `clientSecret`: reCAPTCHA secret key. If you do not have your project ID at hand, check the Google Cloud documentation to learn how to [locate your project ID](https://support.google.com/googleapi/answer/7014113)
```
Comment (GusGuerra) [inline docs/guides/Integration-Guides/recaptcha/implementing-recaptcha-in-integrations.md:49]: Essa prop corresponde a API Key

File: docs/guides/Integration-Guides/recaptcha/implementing-recaptcha-in-integrations.md [hunk 2]
```diff
@@ -3,22 +3,73 @@ title: "Implementing reCAPTCHA in integrations"
...
+- `label`: use this field to identify your key. We recommend that you indicate the mobile platform in which you intend to use the key (e.g., `android-001`).
+- `score`: [reCAPTCHA v3 score](https://developers.vtex.com/docs/guides/recaptcha#recaptcha-v3-score) (optional).
+
```
Comment (GusGuerra) [inline docs/guides/Integration-Guides/recaptcha/implementing-recaptcha-in-integrations.md:52]: Faltou descrever a propriedade `projectId`. Ficou meio misturada com o `clientSecret`

Comment (PedroAntunesCosta) [reply docs/guides/Integration-Guides/recaptcha/implementing-recaptcha-in-integrations.md:53]: Obrigado pela dica! Foi um bug mental aqui mesmo.

**Review by GusGuerra — APPROVED** [review]

File: docs/guides/Integration-Guides/recaptcha/implementing-recaptcha-in-integrations.md [hunk 3]
```diff
@@ -3,22 +3,74 @@ title: "Implementing reCAPTCHA in integrations"
...
-updatedAt: "2022-09-05T16:48:05.147Z"
+updatedAt: "2023-06-16T10:20:43.993Z"
 ---
-In order to have your implementation validate users with reCAPTCHA, they should first obtain the appropriate key from VTEX’s Checkout API and then use that key to display the actual reCAPTCHA widget on their UI. Once the shopper correctly solves the validation, the integration receives a token that should be sent in the request that places the order.
 
->⚠️ You must have a [Beta environment](https://help.vtex.com/pt/tutorial/acessar-o-ambiente-beta-pelo-dominio-myvtex-com--3BHM289568gcSwk2O80Asu) and use [Checkout V6](https://help.vtex.com/pt/tutorial/ativar-o-checkout-v6--7qVqv3ptRvpVVplrvg8ruH) to test the feature.
+If your store uses integrations to place orders, such as mobile apps or proprietary web-based storefronts, follow the steps below to implement [reCAPTCHA validation](https://developers.vtex.com/docs/guides/recaptcha).
 
+>⚠️ You must have a [Beta environment](https://help.vtex.com/pt/tutorial/acessar-o-ambiente-pelo-dominio-myvtex-com--3BHM289568gcSwk2O80Asu) and use [Checkout V6](https://help.vtex.com/pt/tutorial/ativar-o-checkout-v6--7qVqv3ptRvpVVplrvg8ruH) to test the feature.
```
Comment (georgebrindeiro) [inline docs/guides/Integration-Guides/recaptcha/implementing-recaptcha-in-integrations.md]: Probably not relevant/useful anymore. Removing it!

**Review by georgebrindeiro — APPROVED** [review]

**Review by brunoamui — APPROVED** [review]
LGTM

**Pull request merged**